tools/kmod_scripts/sof_remove.sh

```diff
@@ -44,13 +44,16 @@ remove_module snd_soc_rt5682
 remove_module snd_soc_rl6231
 remove_module snd_soc_rt274
 remove_module snd_soc_da7213
+remove_module snd_soc_sst_bxt_da7219_max98357a
```

Comment:
can you move this one up to line 35, this is a machine driver and it belongs with the other machine drivers. Let's try to keep things tidy by and not mix machine and codec drivers.

Reply:
Sure, I will update it.